test/common/filesystem/filesystem_impl_test.cc

```diff
@@ -12,6 +12,10 @@
 namespace Envoy {
 namespace Filesystem {
 
+static constexpr FlagSet DEFAULT_FLAGS{
```

Comment:
Please use the default flags function that is already defined, same elsewhere (I already commented on this I think).

Reply:
I'm not sure I can use the default function which is defined in `access_log_manager_impl.h`. `file_system_test` should not dependent on other implementations .